### ISSUE TITLE
How to use GPU acceleration？

### ISSUE THREAD

**Issue by codybai-19** [KichangKim/DeepDanbooru#109]
My training has started successfully, but it seems that I am only using the CPU and the speed is very slow.

(tf_env) E:\tf_env\DeepDanbooru>deepdanbooru train-project E:\tf_env\DeepDanbooru\deepdanbooru\Myproject
Using SGD optimizer ...
Loading tags ...
Creating model (resnet_custom_v3) ...
Model : (None, 512, 512, 3) -> (None, 20)
Using loss : binary_crossentropy
Loading database ...
No checkpoint. Starting new training ... (2025-02-13 04:46:37.252545)
Shuffling samples (epoch 0) ...
Trying to change learning rate to 5.0 ...
Learning rate is changed to 5 ...
Epoch[0] Loss=2.117366, P=0.368132, R=0.285106, F1=0.321343, Speed = 0.9 samples/s, 1.09 %, ETA = 2025-02-13 10:43:35
Epoch[0] Loss=0.260852, P=0.464539, R=0.280514, F1=0.349800, Speed = 1.0 samples/s, 2.19 %, ETA = 2025-02-13 10:05:38
Saving checkpoint ... (2025-02-13 04:56:56.185561)
Shuffling samples (epoch 1) ...
Trying to change learning rate to 5.0 ...
Learning rate is changed to 5 ...
Epoch[1] Loss=0.306973, P=0.395349, R=0.245868, F1=0.303185, Speed = 0.9 samples/s, 3.28 %, ETA = 2025-02-13 10:11:36
Epoch[1] Loss=0.274003, P=0.457386, R=0.335417, F1=0.387019, Speed = 1.0 samples/s, 4.37 %, ETA = 2025-02-13 10:03:16
Epoch[1] Loss=0.303950, P=0.397306, R=0.251599, F1=0.308094, Speed = 1.0 samples/s, 5.46 %, ETA = 2025-02-13 10:03:35
Saving checkpoint ... (2025-02-13 05:06:56.823303)
